assets/prometheus-user-workload/prometheus.yaml

```diff
@@ -8,18 +8,16 @@ metadata:
 spec:
   affinity:
     podAntiAffinity:
-      preferredDuringSchedulingIgnoredDuringExecution:
-      - podAffinityTerm:
-          labelSelector:
-            matchExpressions:
-            - key: prometheus
-              operator: In
-              values:
-              - k8s
```

Comment:
:thinking: so there was a bug in 4.7?

Reply:
yes the anti-affinity constraints weren't matching the user-workload Prometheus